shared/constants/tracker.js

```diff
@@ -36,6 +39,7 @@ export const updateProofState = 'tracker:updateProofState'
 export const reportLastTrack = 'tracker:reportLastTrack'
 
 export const onClose = 'tracker:onClose'
+export type OnClose = TypedAction<'tracker:onClose', void, void>
```

Comment:
Style question: Can you use `onClose` here to avoid repeating the constant? Worth doing?